### ISSUE TITLE
Starting a transaction fails before PIN screen is reached. Turning accessibility off and then running a transaction fixes it

### ISSUE THREAD

**Issue by davkutalek** [UseHover/SDK-Issues#12]

**Comment by davkutalek** [comment]
Believed fixed in 1.1.+
